app/api/sessions.py

```diff
@@ -86,6 +81,13 @@ def after_create_object(self, session, data, view_kwargs):
                 speaker_id=speaker.id,
             )
             save_to_db(session_speaker_link, "Session Speaker Link Saved")
+        
+        if session.event.get_owner():
```

Comment:
Can we remove these lines of code from here ?

Reply:
No, they are invoked when organiser creates session from event dashboard and adds the speaker from there.